### PR TITLE
Add option to configure rvm branch

### PR DESCRIPTION
**Pull request by akofink**
RVM's stable branch is broken at the time of opening this PR, but the master/head branch works fine.

### CODE REVIEW

**Comment by ekohl** [comment]
Don't you want a parameter in ```init.pp``` so you can actually switch it?

**Comment by akofink** [comment]
Updated. Pardon my lack of puppet knowledge. :apple:

File: manifests/init.pp [hunk 1]
```diff
@@ -54,6 +56,7 @@
   Enum['postgres', 'sqlite'] $db_type = $katello_devel::params::db_type,
   Boolean $use_rvm = $katello_devel::params::use_rvm,
   String $rvm_ruby = $katello_devel::params::rvm_ruby,
+  String $rvm_branch = $katello_devel::params::rvm_branch,
```
Comment (ekohl) [inline manifests/init.pp]: This works. If it's only stable or head you can enforce it with ```Enum['stable', 'head']``` as well.

**Comment by akofink** [comment]
I don't think we should change the type.

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**